Translation/Extractor/File/TwigFileExtractor.php

```diff
@@ -70,10 +80,8 @@ public function enterNode(\Twig_NodeInterface $node, \Twig_Environment $env)
                 $arguments = $node->getNode('arguments');
                 if ($arguments->hasNode($index)) {
                     $argument = $arguments->getNode($index);
-                    if (!$argument instanceof \Twig_Node_Expression_Constant) {
+                    if (!$this->checkNodeIsConstant($argument)) {
                         return $node;
-                        // FIXME: Throw exception if there is some way for the user to turn this off
-                        //        on a case-by-case basis, similar to @Ignore in PHP
```

Comment:
You cant remove this comment if you do not comply with the FIXME. It states that a user should be able to "*turn this off on a case-by-case basis, similar to `@Ignore` in PHP*".